src/Main.cc

```diff
@@ -519,6 +520,27 @@ void ExitBackend(int s) {
     exit(0);
 }
 
+bool FindExecutablePath(std::string& path) {
+    char path_buffer[PATH_MAX + 1];
+#ifdef __APPLE__
+    uint32_t len = sizeof(path_buffer);
+
+    if (_NSGetExecutablePath(path_buffer, &len) != 0) {
```

Comment:
To get this to work on MacOS, you need to add the following (between lines 14 and 15):

```
#ifdef __APPLE__
#include <mach-o/dyld.h>
#endif
```

Reply:
I will add this.